src/app.service.js

```diff
@@ -67,10 +80,20 @@ export class AppService {
     while (stack.length > 0) {
       const currentElement = stack.pop();
       const tagName = currentElement.tagName.toLowerCase();
+      const { className } = currentElement;
+      const { id } = currentElement;
```

Comment:
```suggestion
      const { className, id } = currentElement;
```
한 줄로 줄일 수 있을 것 같습니다!